modules/accredited_representative_portal/app/models/accredited_representative_portal/power_of_attorney_request.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+  class PowerOfAttorneyRequest < ApplicationRecord
+    belongs_to :claimant,
+               class_name: 'UserAccount'
+
+    has_one :form,
+            class_name: 'AccreditedRepresentativePortal::PowerOfAttorneyForm',
+            inverse_of: :power_of_attorney_request,
+            dependent: :destroy
```

Comment:
I think the main thrust of letting active record do cascades is to leverage model lifecycle callbacks. I am pretty tempted to say that we could update the foreign key definitions with cascade behavior so that this happens in the DB. Also I think it's reasonable to heavily avoid model lifecycle callbacks other than something like `after_initialize` for calculated defaults or something like this. I think our (eventual/soon) service layer wants to do this proper business logic. cc @chumpy 

So yeah, maybe we are very justified with 1. DB handles cascade 2. avoid most model lifecycle callbacks.

Reply:
interesting!